validation/schemas/signup.js

```diff
@@ -1,3 +1,7 @@
+const {
+  passwords: { MIN_LENGTH, MAX_LENGTH }
+} = require('~/consts/utils')
+
 const signupValidationSchema = {
   firstName: {
```

Comment:
add regex and length va;idation for firstname

Reply:
same for lastname

Reply:
add role validation
correct values: admin, student, mentor

Reply:
considering there are no validation helpers available for regex and enum validation, will it be okay if I create them myself?

Reply:
Yep, create validateRegex and validateEnum